tests/mxnet/test_nn.py

```diff
@@ -248,52 +263,70 @@ def test_dense_cheb_conv():
 
 def test_dense_graph_conv():
     ctx = F.ctx()
-    g = dgl.DGLGraph(sp.sparse.random(100, 100, density=0.3), readonly=True)
-    adj = g.adjacency_matrix(ctx=ctx).tostype('default')
-    conv = nn.GraphConv(5, 2, norm='none', bias=True)
-    dense_conv = nn.DenseGraphConv(5, 2, norm=False, bias=True)
-    conv.initialize(ctx=ctx)
-    dense_conv.initialize(ctx=ctx)
-    dense_conv.weight.set_data(
-        conv.weight.data())
-    dense_conv.bias.set_data(
-        conv.bias.data())
-    feat = F.randn((100, 5))
-
-    out_conv = conv(g, feat)
-    out_dense_conv = dense_conv(adj, feat)
-    assert F.allclose(out_conv, out_dense_conv)
+    for norm_type in ['both', 'right', 'none']:
+        for g in [
+            dgl.graph(nx.erdos_renyi_graph(100, 0.3)),
+            dgl.bipartite(sp.sparse.random(100, 200, density=0.1))
+        ]:
+            adj = g.adjacency_matrix(ctx=ctx).tostype('default')
+            conv = nn.GraphConv(5, 2, norm=norm_type, bias=True)
+            dense_conv = nn.DenseGraphConv(5, 2, norm=norm_type, bias=True)
+            conv.initialize(ctx=ctx)
+            dense_conv.initialize(ctx=ctx)
+            dense_conv.weight.set_data(
+                conv.weight.data())
+            dense_conv.bias.set_data(
+                conv.bias.data())
+            feat = F.randn((g.number_of_src_nodes(), 5))
+            out_conv = conv(g, feat)
+            out_dense_conv = dense_conv(adj, feat)
+            assert F.allclose(out_conv, out_dense_conv)
 
 def test_dense_sage_conv():
     ctx = F.ctx()
-    g = dgl.DGLGraph(sp.sparse.random(100, 100, density=0.1), readonly=True)
-    adj = g.adjacency_matrix(ctx=ctx).tostype('default')
-    sage = nn.SAGEConv(5, 2, 'gcn')
-    dense_sage = nn.DenseSAGEConv(5, 2)
-    sage.initialize(ctx=ctx)
-    dense_sage.initialize(ctx=ctx)
-    dense_sage.fc.weight.set_data(
-        sage.fc_neigh.weight.data())
-    dense_sage.fc.bias.set_data(
-        sage.fc_neigh.bias.data())
-    feat = F.randn((100, 5))
-
-    out_sage = sage(g, feat)
-    out_dense_sage = dense_sage(adj, feat)
-    assert F.allclose(out_sage, out_dense_sage)
+    for g in [
+        dgl.graph(sp.sparse.random(100, 100, density=0.1)),
+        dgl.bipartite(sp.sparse.random(100, 200, density=0.1))]:
```

Comment:
Same here. You could put the graph you'd like to test here: https://github.com/dmlc/dgl/blob/master/tests/test_utils/graph_cases.py